group_project_v2/mixins.py

```diff
@@ -104,12 +104,7 @@ def user_id(self):
 
     @lazy
     def user_preferences(self):
```

Comment:
@jbzdak actually, `whole lifespan of the Xblock object` should be shorter than default `memoize_with_expiration` time span, so it should not be an issue. Preferences are not supposed to change during single HTTP request that renders GPv2, so it shouldn't be an issue anyway.